Automattic/pocket-casts-ios
# Update protobuf files for Settings sync

Fixes #1404

Just updates the protobuf files to latest with new NamedSettings values and types.

## To test

🟢 CI should be green

1. Run the app
2. Login
3. Subscribe to podcast and make sure syncing works

## Checklist
